### PR TITLE
lv_conf_checker: Update lv_conf_checker so it is ESP-IDF aware.

### PR DESCRIPTION
**Pull request by C47D**
This is part of the work being done to make LVGL an ESP-IDF component. I've decided to break the changes into smaller PRs so they can be easier to review.

Include ESP-IDF headers and lv_conf_kconfig.h when using the ESP-IDF framework. Also remove the CONFIG_LV_CONF_SKIP as it is not generated by the Kconfig file.

NOTE: I'm a bit confused against what branch should the PR be done, please let me know if I should make it against `dev` or `dev-v8` instead, got more PRs coming.

### CODE REVIEW

File: scripts/lv_conf_checker.py [hunk 1]
```diff
@@ -27,13 +27,16 @@
...
 
-/*Handle special Kconfig options*/
-#include "lv_conf_kconfig.h"
-
 #include <stdint.h>
 
+#if defined (ESP_PLATFORM)
+#include "sdkconfig.h"
+#include "esp_attr.h"
+#include "lv_conf_kconfig.h"
```
Comment (embeddedt) [inline scripts/lv_conf_checker.py:39]: Shouldn't `lv_conf_kconfig.h` be included in all cases? ESP-IDF is not the only framework we are targeting for this.

**Comment by embeddedt** [comment]
It's fine for this time, because changing the branch from `master` to `dev` is easy, but could you send it to `dev` next time, please?

**Comment by C47D** [comment]
Sure, I will send it to dev. Please let me know if you need me to make any changes.

File: scripts/lv_conf_checker.py (hunk 1 above)
Comment (C47D) [reply scripts/lv_conf_checker.py:39]: You're right, I was just being safe to don't break anything else by accident, will update the PR.

Comment (C47D) [reply scripts/lv_conf_checker.py:39]: Fixed with c3698d6

**Comment by kisvegabor** [comment]
Thank you, Carlos!

**Pull request merged**